benchmarks/hashes.txt

```diff
@@ -0,0 +1,32 @@
+04468ad216555cab2b50dbae62b57f56bf694cc3
+a964aebfb904d5ac4cb2645478f38189b27ad97a
+7f77dabbfcd1be8b5943cbb6ee3b049c66d337f6
+d78cc94a9538aeb91b30494bdcde1348cab0e2f6
+040b01283470b95d23a244eb1a791dfd189502c6
+5ca84aca6002ec1550417bf791b3f8cc9e551818
+2dc133ef61baf3f6b2b54814cbeb3299282ee794
+1d3f26bfcabab7751bd324d1fd48825667c2e949
+126e00f691007ff71e6a2c327b19f92d8a478adb
+2c0c4ee67f732b0d1b7530e8363b15c57b936b97
+619b95ee13f82560c3cd596095bcd606ba0277ae
+6e18639b3db4121322e0fc7009df463b8dc7ee9f
+fe0a64b270dbde0946fdcb0819aad178d42f77e6
+6aea7d4aaf424e2a514c004f99b271ecfb446237
+c48c441ec6b8919f104edd6affb57b69aa275a38
+7d9837ead056af2e36009a0b0ed8d1a9e9bf3947
+e7e7de5ca2e1c19963be8a918369fb19186f9a73
+f65bebb6930fdd5ec891a905c30489a52add05f6
+a171cdfbab64903ae67d7461b0904f350191ee29
+7196ae6794a5f7b3ab0a48e880f1797a218e59de
+5ee3ff950709ab356d7349d6b56818b0c5fe624b
+697834d17d6fb5bb5d0adf945a5d5f5d187506a1
+2b296e79f0260b8d547af8d078178f079d90a3d9
+a91762071e3cecc8d995c993a3175f84ed9ec804
+82b2c2f97b3f929568a07850de4f75b0d16f3c56
+3d713cf920c1f164c1c3c2fcfab96956467b0d64
+eed85601c8bdf0829ed256cdf2af9982bc9ed5a2
+4e6dbff740292ae57230686ad4417e46d63e171c
+222a1c89e9ce5c24debc749558cf5eb3b231f89a
+85603e644e9ba004188ef79e499b9a40557b446c
+9a05e600275c592451bda9287205fe0e99a87bb3
+6b0b531d5311ab0f61aefd3e76a8dec9144d6b81
```

Comment:
What's this file about?

Reply:
This should be removed, thanks for letting me know